### PR TITLE
chore: added multiple links

### LINKED ISSUE #640: [OTHER] adding multiple links
### What would you like to share or ask?

Hey, I want to add some links in these:
- projects sub-category under OpenSource category
- python,  javascript, C# sub-category under Languages category
- blogs, e-book, hosting, project ideas sub-category in resources category
- android, computer science subcategory in youtube category
- ReactJS subcategory in frontend category

### Checklist

- [X] I have checked the existing issues
- [X] I have read the [Contributing Guidelines](https://github.com/rupali-codes/LinksHub/blob/main/CONTRIBUTING.md)
- [X] I am willing to work on this issue (optional)
- [X] I am a GSSoC'23 contributor

### PR DESCRIPTION
**Pull request by akkshayTandon**
## Fixes Issue

Closes #640 

## Changes proposed

added multiple links as per mentioned in the issue

## Screenshots

nil

## Note to reviewers

nil

### CODE REVIEW

File: database/frontend/online-code-editors.json [hunk 1]
```diff
@@ -82,5 +82,12 @@
     "url": "https://leetcode.com/",
     "category": "frontend",
     "subcategory": "online-code-editors"
+  },
+  {
+    "name": "Programiz",
+    "description": "Learn to program with our beginner-friendly tutorials and examples. Read tutorials, try examples, write code and learn to program.",
```
Comment (CBID2) [inline database/frontend/online-code-editors.json:88]: [Proofreading]
```suggestion
    "description": "Learn to program with our beginner-friendly tutorials and examples. Read tutorials, try examples, write code, and learn to program.",
```

**Review by CBID2 — CHANGES_REQUESTED** [review]
Hi, @akkshayTandon! :) Aside from the proofreading error I mentioned in my inline comment, you're off to a great start! :) Keep it up! :)

**Review by CBID2 — APPROVED** [review]
Looks great @akkshayTandon! :) LGTM! :)

**Comment by rupali-codes** [comment]
@akkshayTandon hey thanks, could you please conflicts? (NOTE - please don't close the PR, you can learn to resolve conflicts [here](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/addressing-merge-conflicts/resolving-a-merge-conflict-on-github)

**Review by CBID2 — DISMISSED** [review]
Great job @akkshayTandon! 😄 LGTM! 😄

File: database/frontend/react.json [hunk 2]
```diff
@@ -6,5 +6,21 @@
...
         "subcategory":"react",
         "language":"english"
+    },
+    {
+        "name":"React.dev",
+        "description":"the official documentation of ReactJS",
+        "url":"https://react.dev/learn",
+        "category":"frontend",
+        "subcategory":"react",
+        "language":"english"
```
Comment (rupali-codes) [inline database/frontend/react.json:16]:
```suggestion
```

File: database/frontend/react.json [hunk 3]
```diff
@@ -6,5 +6,21 @@
...
+    },
+    {
+        "name":"30 days of React",
+        "description":"a very nice repository and a challenge at the same time to learn ReactJS",
+        "url":"https://github.com/Asabeneh/30-Days-Of-React",
+        "category":"frontend",
+        "subcategory":"react",
+        "language":"english"
```
Comment (rupali-codes) [inline database/frontend/react.json:24]:
```suggestion
```

File: database/languages/csharp.json [hunk 4]
```diff
@@ -5,5 +5,19 @@
         "url": "https://www.tutorialsteacher.com/csharp/",
         "category": "Languages",
         "subcategory": "csharp (C#)"
+    },
+    {
+        "name": "Introduction to C#",
+        "description": "A beginner friendly interactive tutorial to C# by Microsoft itself.",
+        "url": "https://learn.microsoft.com/en-us/dotnet/csharp/tour-of-csharp/tutorials/",
+        "category": "Languages",
```
Comment (rupali-codes) [inline database/languages/csharp.json:13]:
```suggestion
        "category": "languages",
```

File: database/languages/csharp.json [hunk 5]
```diff
@@ -5,5 +5,19 @@
...
+        "subcategory": "csharp (C#)"
+    },
+    {
+        "name": "Programiz",
+        "description": "A nice and beginner friendly tutorial for C#.",
+        "url": "https://www.programiz.com/csharp-programming",
+        "category": "Languages",
```
Comment (rupali-codes) [inline database/languages/csharp.json:20]:
```suggestion
        "category": "languages",
```

File: database/languages/python.json
Comment (rupali-codes) [inline database/languages/python.json:1]: this category is to ad Python tutorials such as blog posts or articles, for adding a youtube channel, please create a new subcategory named **python-youtube** . let us know if you any kinda help

File: database/opensource/projects.json [hunk 6]
```diff
@@ -17,8 +17,8 @@
     "name": "AsaKatsu",
     "description": "'AsaKatsu' is a Japanese term for waking up a little early to do something productive, aligned with a specific goal. With this philosophy in mind, this product helps you keep track of your goals' progress.",
     "url": "https://github.com/asakatsuOrg/AsaKatsuProject",
-    "category": "open source",
-    "subcategory": "open-source-projects"
+    "category": "opensource",
```
Comment (rupali-codes) [inline database/opensource/projects.json:20]:
```suggestion
    "category": "open source",
```

File: database/opensource/projects.json [hunk 7]
```diff
@@ -17,8 +17,8 @@
...
+    "subcategory": "projects"
```
Comment (rupali-codes) [inline database/opensource/projects.json:21]:
```suggestion
    "subcategory": "open-source-projects"
```

**Comment by rupali-codes** [comment]
@akkshayTandon could you please make the suggested changes and resolve merge conflicts?

**Comment by rupali-codes** [comment]
@akkshayTandon are you still working on this?

**Comment by akkshayTandon** [comment]
@rupali-codes yep, actually i was busy in another event(khelo india) so didn't got the time to see this

**Review by CBID2 — APPROVED** [review]
LGTM!

**Review by rupali-codes — APPROVED** [review]
lgtm too

**Pull request merged**